src/systemd/sd-event.h

```diff
@@ -146,6 +146,12 @@ int sd_event_source_get_destroy_callback(sd_event_source *s, sd_event_destroy_t
 int sd_event_source_get_floating(sd_event_source *s);
 int sd_event_source_set_floating(sd_event_source *s, int b);
 
+static __inline__ sd_event_source* sd_event_source_disable_unref(sd_event_source *s) {
```

Comment:
this is a public API, I'd really prefer if we didn't expose inline functions in that as we cannot change what these things do then, ever, as the implementation becomes the API, and that's not cool. Please make this a regular function...